cli/dcoscli/data/help/marathon.txt

```diff
@@ -92,6 +95,15 @@ Commands:
         Display detailed information for a specific pod.
     pod update
         Update a pod.
+    debug list
+        Print a list of currently queued instance launches for
+        debugging purpose.
+    debug summary
+        Display summarized information for a queued instance launch
+        for debugging purpose.
+    debug details
+        Display detailed information for a queued instance launch
+        for debugging purpose.
```

Comment:
It was named `queue` or `launchqueue` before, but the name was declined :)

A queued app is not necessarily related to a deployment. A deployment is always triggered by a user request, but also if a task fails and marathon triggers the relaunch, this app would also land in this queue (without deployment id). Therefore I would not go with `deployment`.